src/shared/style/ancillary/_bootstrap-grid.scss

```diff
@@ -0,0 +1,1002 @@
+.container {
```

Comment:
Don't think we need all of these bootstrap styles. We should only bring styles that are needed for the components we are bringing in.